cobra/cmd/init.go

```diff
@@ -77,3 +78,52 @@ func initializeProject(args []string) (string, error) {
 
 	return project.AbsolutePath, nil
 }
+
+func getModImportPath() string {
+	mod, cd := parseModInfo()
+	return path.Join(mod.Path, fileToURL(strings.TrimPrefix(cd.Dir, mod.Dir)))
+}
+
+func fileToURL(in string) string {
```

Comment:
Is it worth having this function which is called once only? Maybe it can be written inside getModImportPath?

Reply:
Initially I thought I'd use it more than once, though I don't see an issue with leaving it as a separate function. To me it's easier to read this way and performs about the same (not that performance is a big factor here).